travis.yml

```diff
@@ -0,0 +1,15 @@
+language: node_js
+node_js:
+- node
+cache:
+  directories:
+  - node_modules
+deploy:
+  provider: npm
+  email: scheul93@gmail.com
```

Comment:
Did you generate your own Travis key?
I think it uses your email as part of the encryption